testutils/inject/base.go

```diff
@@ -118,3 +122,19 @@ func (b *Base) Geometries(ctx context.Context, extra map[string]interface{}) ([]
 	}
 	return b.GeometriesFunc(ctx)
 }
+
+// CurrentInputs returns the base's current inputs.
+func (b *Base) CurrentInputs(ctx context.Context) ([]referenceframe.Input, error) {
```

Comment:
We can't add this here as it is not a part of the Base interface.  We have functionality availible for doing similar things in the kinematicbase package

Reply:
🤔 hmmmm ok, I will look into this, I know that we have a fake kinematic base, but I did not now we have an injected kinematic base.
 
The reason I made these changes were for `TestStoppableMoveFunction`.

Within these tests, specifically for MoveOnMap and MoveOnGlobe, we rely on an injected base since we want to define our own behavior for functions. 

For this reason, I think we should remain using an injected struct.

Reply:
Well we can't add these functions to the inject/base package so we will have to find another way to test then.  We can talk about strategy for this later.  I think there are some issues we have to work out in the non-test portion of this PR, so figure those out first

Reply:
what about an injected `wheeled.Base`?